frontend/src/CardGallery/components/Chat/UserChat.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect, useState} from "react";
+import {useParams} from "react-router-dom";
+import {ChatMessage, NewChatMessage} from "../../models/ChatMessage";
+
+export default function UserChat(){
+
+    const [messages, setMessages] = useState<ChatMessage[]>([]);
+    const [message, setMessage] = useState('');
+    const [ws, setWs] = useState<WebSocket | null>(null);
+    const { senderUsername, receiverUsername } = useParams<{ senderUsername: string, receiverUsername: string }>();
+
+    console.log(messages);
+    useEffect(() => {
+        const ws = new WebSocket(`ws://localhost:8080/api/ws/chat`);
+        setWs(ws);
+
+        ws.onmessage = (event) => {
```

Comment:
What type of Event? It can be any type I guess?

Reply:
ja ich glaube schon